test/unittests/iotools/converters/ixi_to_bids/test_ixi_to_bids_utils.py

```diff
@@ -0,0 +1,614 @@
+import json
+from asyncore import write
```

Comment:
This is breaking the unit test suite for Python 3.12 and I don't believe you added this intentionally. Looks more like a little joke from PyCharm... ;)

```suggestion
```

Reply:
Yup I confirm !